docs/trex_sandbox_test.md

```diff
@@ -0,0 +1,201 @@
+</div>
+
+<!-- 
+Sandboxed Extensions are a type of Tableau Extension that are not permitted to make external network requests and thus cannot exfiltrate end-user data. A Sandboxed Extension is hosted by Tableau and, using W3C standards, is wrapped inside an environment which prevents communication to anything except the Tableau hosting server. For further details, see Tableau Extensions API.  -->
+
+
+
+**In this section**
+
+* TOC
+{:toc}
+
+
+---
+
+
+## Sandboxed Extensions and Network Enabled Extensions
+
+Tableau supports two types of dashboard extensions:
+
+* **Network Enabled Extensions** - can access resources and applications outside of Tableau. Supported in Tableau 2018.2 and later.
+
+* **Sandboxed Extensions** - run in a Tableau hosted environment and cannot make network calls. Supported in Tableau 2019.4 and later. Available for testing with Tableau 2019.3.
+
+
+## Create Sandboxed Extensions
+
+You can create Sandboxed Extensions from scratch, or you can port your existing (Network Enabled) extensions to work in the sandbox environment. The components of a Sandboxed Extension are the same as for a Network Enabled one. Each extension consists of a web page that calls your JavaScript code and the Extensions API JavaScript library. Each extension provides a `.trex` file.
+
+To port an existing extension, make sure all resources and libraries are local to your extension source code. No network access is allowed. And you need to modify the path to your extension in the `.trex` file to have it load in the sandbox environment. Information about changing your `.trex` file is described in [Test your own Sandboxed Extensions in the local sandbox](#test-your-own-sandboxed-extensions-in-the-local-sandbox).
+
+
+### Requirements for Sandboxed Extensions
+
+* Sandboxed Extensions cannot make calls to remote or external external resources or libraries. All content must be local to the extension. For example, if your extension uses the JQuery library, the library can't be accessed via HTTPS (`src="https://code.jquery.com/jquery-3.2.1.min.js"`). The library must be downloaded and on the same computer as the extension. All links must be relative (`src="./jquery-3.2.1.min.js`).
+
+* All resources must be static files (`.js`, `.css`, images, etc.).
+
+* Resources should be placed in the same folder as your extension web page(s), or in a sub-folder of that directory. All references must be relative.
+
+* No inline scripts or styles. Follow the guidelines that all extensions should follow to reduce XSS vulnerabilities, see [Cross-Site Scripting and Extension Security]({{site.baseure}}/docs/trex_xss_guidance.html).
```

Comment:
This is not a requirement. This is a suggestion. Inline scripts and styles are allowed.

Reply:
Oh.. I'll change that.  I'll drop inline scripts, styles.  -  "Follow the suggested guidelines....."